doc/tutorials/test-templates-basic.md

```diff
@@ -0,0 +1,306 @@
+# Creating Tests with JSonnet Templates
+
+## Prerequisites
+
+1. A clone of this repository.
+1. Build dependencies from [developing.md](/doc/developing.md).
```

Comment:
another prereq is a GCP project. Maybe link to some page on setting up a GCP project

Reply:
Done.